conference/src/components/CreateSubmissionForm/CreateSubmissionForm.tsx

```diff
@@ -0,0 +1,114 @@
+import { useFormik } from "formik";
+import { ChangeEvent, useEffect } from "react";
+import { useNavigate, useParams } from "react-router-dom";
+import UploadFile from "@mui/icons-material/UploadFile";
+import Alert from "@mui/material/Alert";
+import Box from "@mui/material/Box";
+import Button from "@mui/material/Button";
+import Collapse from "@mui/material/Collapse";
+import FormControl from "@mui/material/FormControl";
+import FormHelperText from "@mui/material/FormHelperText";
+import TextField from "@mui/material/TextField";
+import { usePostCreateSubmissionApi } from "./CreateSubmissionForm.hooks";
+import { initialValues } from "./CreateSubmissionForm.types";
+import { validationSchema } from "./CreateSubmissionForm.validator";
+
+export const CreateSubmissionForm = () => {
+  const navigate = useNavigate();
+  const { conferenceId } = useParams();
+  const { response, post } = usePostCreateSubmissionApi();
+
+  useEffect(() => {
+    if (!response.isLoading && !response.isError && response.data) {
+      navigate(`/conferences/${conferenceId}/submissions/${response.data.id}`);
+    }
+  }, [response, navigate, conferenceId]);
+
+  const formik = useFormik({
+    initialValues: { ...initialValues, conferenceId: Number(conferenceId) },
+    validationSchema: validationSchema,
+    onSubmit: (values) => {
+      const formData = new FormData();
+      for (const [field, value] of Object.entries(values)) {
+        if (value !== null) {
```

Comment:
validator will not allow nulls, i guess we can remove this check.
i would also recommend moving this logic to some util function and just reuse here